tests/unit/samplegen/common_types.py

```diff
@@ -17,6 +17,8 @@
 from collections import namedtuple
 from typing import(Iterable, Optional)
 
+from google.protobuf import descriptor_pb2
```

Comment:
Should it be `descriptor_pb2` or `descriptor`?

Reply:
Other code in the microgenerator uses `descriptor_pb2`.